Classes/Module/Mailhog.php

```diff
@@ -77,6 +77,50 @@ public function followLinkInTheEmail($link)
         throw new \Exception(sprintf('Did not find the link "%s" in the mail', $link));
     }
 
+    /**
+     * @param $text
+     * @throws \Exception
+     */
+    public function seeTextInMail($text)
+    {
+        $mail = $this->parseMailBody($this->currentMail->getBody());
+        if (stristr($mail, $text)) {
+            return;
+        }
+        throw new \Exception(sprintf('Did not find the text "%s" in the mail', $text));
+    }
+
+    /**
+     * @param $address
+     * @throws \Exception
+     */
+    public function checkRecipientAddress($address)
```

Comment:
Hey @kabarakh, all the method header are missing parameter and return types. Aren't they possible here? If they are possible, please add them.